biorxiv/article_distances/03_biorxiv_corrected_published_rate.py

```diff
@@ -185,9 +185,9 @@
     .groupby("pub_month")
```

Comment:
Is there a reason you're replacing the month here? Maybe add a comment would be helpful here

Reply:
So a small handful of preprints were "posted" during October of 2013; however bioRxiv didn't really start until November 2013. So I'm updating their posting to be concurrent with bioRxiv's true start month. Adding a comment to reflect this point.